compliance_controller/compliance_controller/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(compliance_controller)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin
+  REQUIRED COMPONENTS
+    compliance_control_msgs
+    message_generation
+    realtime_tools
+    roscpp
+    std_srvs
+    traj_or_jog_controller
+)
+
+find_package(traj_or_jog_controller REQUIRED)
+
+catkin_package(
+  CATKIN_DEPENDS
+    compliance_control_msgs
+    realtime_tools
+    roscpp
+    traj_or_jog_controller
+  INCLUDE_DIRS include
+  LIBRARIES compliance_controller traj_or_jog_controller
+)
+
+include_directories(include ${catkin_INCLUDE_DIRS} ${traj_or_jog_controller_INCLUDE_DIRS} ${Boost_INCLUDE_DIR})
+
+add_library(compliance_controller src/compliance_controller.cpp)
+target_link_libraries(compliance_controller ${catkin_LIBRARIES})
+
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
+
```

Comment:
We proud ourselves here for our tests. They ensure that continued improvements to the controllers and the ros_control framework keep everything working. Could you please add some rostests that ensure some basic functionality? Look at joint traj controller or diff drive controller for good examples.